skins/laika/src/scss/overrides.scss

```diff
@@ -85,6 +111,14 @@ button.is-main {
 	}
 }
 
+@include from($tablet) {
```

Comment:
I did it this way in order to keep all the rules for a certain view all together following the other `include tablet` rule. I am fine with the other grouping as well but this is a matter of preference I guess. Should the other one be grouped with the `button.is-main` rules then?
```
@include until($tablet) {
	button.is-main {
		width: 100%;
	}
}
```

Reply:
I also have not made up my mind on what is actually better, I think both approaches have pros and cons, I think I have a slight preference for my suggestion because it makes sure you always see all relevant rules for one element rather than having to go CTRL + F yourself through a file. Again though like the other thing, it's not a blocking issue, just wanted to comment on it.